cron/hourly/20-fix-stripe-stale-NEW-paymentIntents.ts

```diff
@@ -0,0 +1,54 @@
+import '../../server/env';
+
+import moment from 'moment';
+
+import OrderStatuses from '../../server/constants/order-status';
+import { PAYMENT_METHOD_SERVICE, PAYMENT_METHOD_TYPE } from '../../server/constants/paymentMethods';
+import logger from '../../server/lib/logger';
+import { syncOrder } from '../../server/lib/stripe/sync-order';
+import { Collective, Op, Order, PaymentMethod } from '../../server/models';
+import { runCronJob } from '../utils';
+
+async function run() {
+  logger.info('Starting job to fix stripe stale NEW paymentIntents');
+
+  const results = await Order.findAndCountAll({
+    limit: 20,
```

Comment:
We should either remove/increase the limit or implement pagination:
```suggestion
```

Reply:
Added pager, thanks!